cogs/economy.py

```diff
@@ -62,13 +62,12 @@ async def leaderboard(self, ctx):
         embed.set_thumbnail(url="attachment://cactus.png")
 
         sorted_users = self.get_top_ten()
-        for i in range(min(len(sorted_users), 10)):
+        for i in range(min(len(sorted_users), 9)):
```

Comment:
The current implementation isn't showing the 9th person's points, so we might want to reduce this even further.

Reply:
so weird, i'll do 8 then